docs/modules/ROOT/partials/storage-ceph-remove-osd.adoc

```diff
@@ -33,7 +33,7 @@ for node in $(echo -n {osd-replace-list}); do
   kubectl --as=cluster-admin -n syn-rook-ceph-cluster exec -it deploy/rook-ceph-tools -- \
     ceph osd purge "${osd_id}"
   kubectl --as=cluster-admin -n syn-rook-ceph-cluster exec -it deploy/rook-ceph-tools -- \
-    ceph osd crush remove "${osd_id}"
+    ceph osd crush remove "${node}"
```

Comment:
Does this command really use the node id when all others use the osd id? Maybe the command fails because purge already removes the osd from the crushmap?

Reply:
Yes, the OSD is removed from the crush map already with `osd purge`, but the node still remains. With `crush remove` you can remove a failure domain (node for us) from the crush map.